### DISCUSSION TITLE
Problem with project submission

### DISCUSSION THREAD

**Discussion by Diego0106** [microsoft/AI-Chat-App-Hack#121] [Q&A]
I sent the project earlier today but when I saw today's broadcast with the projects, mine did not appear but I did send it as you can see in this screenshot:
<img width="851" alt="image" src="https://github.com/microsoft/AI-Chat-App-Hack/assets/138242708/ca0b7062-626d-4bff-b799-8eeab69796a7">

Then I resubmitted my project with another github account associated with a different email with which I registered. Is there a problem?

In the description I refer to the email with which I registered or is it necessary to register again with the other email

**Comment by pamelafox** [comment]
I didn't see #115 but I do see the most recent one. I'm really not sure what happened to #115, it 404s when I visit it. Since you mention your registered email in the issue, that's fine. Thanks for checking!

**Reply by Diego0106** [reply]
Thank you
